buildSrc/subprojects/plugins/src/main/kotlin/org/gradle/plugins/performance/PerformanceTestPlugin.kt

```diff
@@ -431,15 +431,15 @@ class PerformanceTestPlugin : Plugin<Project> {
     }
 
     private
-    fun Project.configureForAnyDistributedPerformanceTestTask(task: DistributedPerformanceTest) {
-        task.apply {
+    fun PerformanceTest.registerTemplateInputsToPerformanceTest() {
+        apply {
```

Comment:
Isn't the `apply` implicit here, since `PerformanceTest` is already the receiver?